joern-cli/frontends/csharpsrc2cpg/src/main/resources/builtin_types.json

```diff
@@ -4,10 +4,14 @@
       "name": "Console",
```

Comment:
Could you additionally prefix `Console` with `System` while you're editing the file, please? E.g. `System.Console` and `System.ArrayBufferWriter`